[docs] Remove NoSsr where possible (#15510)
mui-org/material-ui: docs/src/pages/lab/slider/slider.md

{{"demo": "pages/lab/slider/VerticalSlider.js"}}

> ## Customized slider

Reviewer: ```suggestion
## Customized sliders
```

If you have been reading the [overrides documentation page](/customization/overrides/)

> but you are not confident jumping in, here are examples of how you can customize the component.

Reviewer: ```suggestion
but you are not confident jumping in, here are some examples of how you can customize the component.
```

{{"demo": "pages/lab/slider/CustomizedSlider.js"}}

> ⚠️ While the material design specification encourages theming, this example is off the beaten path.

Reviewer: ```suggestion
⚠️ While the material design specification encourages theming, these example are off the beaten path.
```
